patroni/dcs/__init__.py

```diff
@@ -819,10 +872,20 @@ def __new__(cls, *args: Any, **kwargs: Any):
             kwargs['workers'] = {}
         return super(Cluster, cls).__new__(cls, *args, **kwargs)
 
+    @property
+    def last_lsn(self) -> int:
+        """Last known leader LSN."""
+        return self.status.last_lsn
+
+    @property
+    def slots(self) -> Optional[Dict[str, int]]:
+        """State of permanent logical replication slots on the primary in the format: ``{"slot_name": int}``."""
```

Comment:
```suggestion
        """State of permanent replication slots on the primary in the format: ``{"slot_name": int}``."""
```

Same comment regarding physical slots.